build.pkr.hcl

```diff
@@ -20,7 +23,7 @@ build {
     environment_vars = [
       "GRAPHDB_VERSION=${var.gdb_version}",
     ]
-    inline      = ["sudo -E bash /tmp/install_graphdb.sh"]
+    inline      = ["sudo -E bash /tmp/install_graphdb.sh", ]
```

Comment:
why the `,`?

Reply:
The block contained additional commands to copy helpers scripts from /tmp to /opt. The additional commands were moved to install_graphdb.sh, and I have missed removing the `,`